### DISCUSSION TITLE
Any plans for binaries for VB2017+

### DISCUSSION THREAD

**Discussion by GaryWayneSmith** [AmpScm/AnkhSVN#16] [General]
Checking to see if there is a plan for binary installers for this project.

**Comment by ARMoir** [comment]
Yes — prebuilt binaries are now available.

Current AnkhSVN releases include a ready-to-install Visual Studio VSIX, so building from source is no longer required for normal installation.

For example, the latest release, **AnkhSVN 2.9.199**, includes:

- `AnkhSVN-2.9.199-VS2022.vsix`
- the corresponding NuGet package
- release/coverage artifacts

The VSIX can be downloaded directly from the GitHub Releases page.

For modern Visual Studio versions, please use the latest release rather than the older 2.9.x binaries that were available when this discussion was originally opened.
